tests/integration/targets/inventory_aws_mq/runme.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -eux
+
+function cleanup() {
+    ansible-playbook playbooks/setup_instance.yml -e "operation=delete" "$@"
+    exit 1
+}
+
+trap 'cleanup "${@}"'  ERR
+
+# ensure test config is empty
+ansible-playbook playbooks/empty_inventory_config.yml "$@"
+
+export ANSIBLE_INVENTORY_ENABLED="community.aws.aws_mq"
+
+# test with default inventory file
+ansible-playbook playbooks/test_invalid_aws_mq_inventory_config.yml "$@"
+
+export ANSIBLE_INVENTORY=test.aws_mq.yml
+
+# test empty inventory config
+ansible-playbook playbooks/test_invalid_aws_mq_inventory_config.yml "$@"
+
+# delete existing resources
+ansible-playbook playbooks/setup_instance.yml -e "operation=delete" "$@"
+
+# generate inventory config and test using it
+ansible-playbook playbooks/create_inventory_config.yml "$@" &&
```

Comment:
```suggestion
ansible-playbook playbooks/create_inventory_config.yml "$@"
```

We're running with `set -e` all of these extra `&&` shouldn't be needed...

Reply:
updated